integration/e2ecortex/client.go

```diff
@@ -391,6 +389,32 @@ type userConfig struct {
 	AlertmanagerConfig string            `yaml:"alertmanager_config"`
 }
 
+// GetAlertmanagerRingStatusPage gets the status page of alertmanager.
+func (c *Client) GetAlertmanagerStatusPage(ctx context.Context) ([]byte, error) {
+	return c.getRawPage(ctx, "/multitenant_alertmanager/status")
+}
+
+func (c *Client) getRawPage(ctx context.Context, url string) ([]byte, error) {
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := c.httpClient.Do(req.WithContext(ctx))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	content, err := io.ReadAll(resp.Body)
```

Comment:
Tests failed because I am using [io.ReadAll](https://golang.org/pkg/io/#ReadAll), I will fix it..

Maybe the CI's go version should be updated to go1.16.

Reply:
> Maybe the CI's go version should be updated to go1.16.

You're definitely right! Going to fix it.